Source/OxyPlot.Wpf.Tests/ExampleLibraryTests.cs

```diff
@@ -9,68 +9,58 @@
 
 namespace OxyPlot.Wpf.Tests
 {
-    using System;
     using System.IO;
-    using System.Linq;
     using NUnit.Framework;
 
     /// <summary>
     /// Provides unit tests for the example library.
     /// </summary>
+    [TestFixture]
     public class ExampleLibraryTests
     {
+        const string DestinationDirectory = "ExampleLibrary.Actual";
+        const string BaselineDirectory = "ExampleLibrary.Baseline";
+        const string DiffDirectory = "ExampleLibrary.Diff";
+
+        [OneTimeSetUp]
+        public void Init()
+        {
+            Directory.CreateDirectory(BaselineDirectory);
+            Directory.CreateDirectory(DestinationDirectory);
+            Directory.CreateDirectory(DiffDirectory);
```

Comment:
Aside: should this delete the existing dirs? Some of the examples use random data so they error if you run the tests multiple times without clearing out the baseline dir. Could also seed the rng in tests but that's not exposed from the examples currently.

Reply:
Baseline should be left alone; the others could be deleted, but it's not a problem I don't think.